flocker/node/agents/ebs.py

```diff
@@ -0,0 +1,229 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+An EBS implementation of the ``IBlockDeviceAPI``.
+"""
+
+from bitmath import Byte, GB
```

Comment:
An unwritten rule (:cry: sorry) is that imports are separated into stdlib, third-party, and Flocker - and should appear in that order.